ts/Core/Renderer/SVG/SVGElement.ts

```diff
@@ -1812,6 +1805,21 @@ class SVGElement implements SVGElementLike {
         element.setAttribute(key, opacity);
     }
 
+    /**
+     * Re-align an aligned text or label after setting the text.
+     *
+     * @private
+     * @function Highcharts.SVGElement#reAlign
+     *
+     */
+    protected reAlign(): void {
+        if (this.alignOptions?.width && this.alignOptions.align !== 'left') {
+            this.alignOptions.width = this.getBBox().width;
+            this.placed = false; // Block animation
```

Comment:
Did you consider adding `animate` argument to `align()`? We have a few places where we could replace `Element.placed = false` with it. I guess `placed` state is a bit more universal: we can set the state in one place, and many operations later we can pick it up in `align()`?

Reply:
I did not consider that. But it sounds like a good idea. Feel free to experiment with it if want.